compass/_scrapers/hierarchy.py

```diff
@@ -0,0 +1,98 @@
+import datetime
+import json
+
+import requests
+from lxml import html
+
+from compass.interface_base import CompassInterfaceBase
+from compass.settings import Settings
+from compass.utility import compass_restify
+
+endpoints = {
+    "Countries": "/countries",
+    "HQ Sections": "/hq/sections",
+    "Regions": "/regions",
+    "Country Sections": "/country/sections",
+    "Counties": "/counties",
+    "Region Sections": "/region/sections",
+    "Districts": "/districts",
+    "County Sections": "/county/sections",
+    "Groups": "/groups",
+    "District Sections": "/district/sections",
+    "Group Sections": "/group/sections",
+}
+
+
+class CompassHierarchyScraper(CompassInterfaceBase):
```

Comment:
Should probably drop the 'Compass' from the class names. Make it more difficult to clearly follow, especially now the scrapers are in their own package.